### ISSUE TITLE
Start without password

### ISSUE THREAD

**Issue by LMech** [pop-os/shell#1234]
**(1) Issue/Bug Description:**
I open and close my laptop frequently and i noticed that sometimes the OS do not ask for the password sometimes and also each time it does not ask for the password the WiFi do not work ( I use automatic login)

**(2) Steps to reproduce (if you know):**

**(3) Expected behavior:**
I think it should ask for the password i power up my laptop and WiFi should work fine.

**(4) Distribution (run `cat /etc/os-release`):**
 OS: Pop!_OS 21.04 x86_64 

**(5) Gnome Shell version:**
GNOME 3.38.4 

**(6) Pop Shell version (run `apt policy pop-shell` or provide the latest commit if building locally):**

```
pop-shell:
  Installed: 1.1.0~1635267414~21.04~1fddaa8
  Candidate: 1.1.0~1635267414~21.04~1fddaa8
  Version table:
 *** 1.1.0~1635267414~21.04~1fddaa8 1001
       1001 http://ppa.launchpad.net/system76/pop/ubuntu hirsute/main amd64 Packages
       1001 http://ppa.launchpad.net/system76/pop/ubuntu hirsute/main i386 Packages
        100 /var/lib/dpkg/status
```

**(7) Where was Pop Shell installed from:**
The official website!!

**(8) Monitor Setup (2 x 1080p, 4K, Primary(Horizontal), Secondary(Vertical), etc):**
1 * 1080p laptop monitor

**(9) Other Installed/Enabled Extensions:**
Slimbook battery is the only thing might interfere with wifi.

**(10) Other Notes:**
If there is a log or something i can send might help i am open to send any (if there is no personal information)

**Comment by jacobgkau** [comment]
If you open the Extensions app (green icon) and disable the Pop Shell extension, does this issue still occur?

**Comment by LMech** [comment]
> If you open the Extensions app (green icon) and disable the Pop Shell extension, does this issue still occur?

I will give it a try because the issue does not happen frequently.

**Comment by jacobgkau** [comment]
> I will give it a try because the issue does not happen frequently.

Please let us know.

**Comment by LMech** [comment]
Yes, i think the problem by the pop shell, after several days i did not faced the same issue (Also the boot is milliseconds slower but not sure it because of the pop shell)

So what is the move now
